src/map/duel.c

```diff
@@ -39,29 +39,12 @@ struct duel_interface *duel;
 /*==========================================
  * Duel organizing functions [LuzZza]
  *------------------------------------------*/
-static void duel_savetime(struct map_session_data *sd)
-{
-	time_t clock;
-	struct tm *t;
-
-	time(&clock);
-	t = localtime(&clock);
-
-	pc_setglobalreg(sd, script->add_variable("PC_LAST_DUEL_TIME"), t->tm_mday*24*60 + t->tm_hour*60 + t->tm_min);
+static void duel_savetime(struct map_session_data* sd) {
+	pc_setglobalreg(sd, script->add_str("PC_LAST_DUEL_TIME"), (int)time(NULL));
```

Comment:
Is there any reason to use `script->add_str()` instead of the previous `script->add_variable()`

note: add_variable marks the string you're adding as a variable name, add_str does not - this means that the variable can't be accessed by a script (unless the script is loaded before this function is called, and the name then gets marked as variable by the script, but that's just a coincidence, and not the intended way for it to work)

Reply:
didn't aware about the different, I was probably copy from somewhere else, for examples.
fixed.